diff --git a/types/Props.d.ts b/types/Props.d.ts
index f8fcd45ab3048d9b34709eba1797597ac2facdec..3afa05a42f485ff23899f20df2ff9d85bf588eed 100644
--- a/types/Props.d.ts
+++ b/types/Props.d.ts
@@ -44,7 +44,7 @@ export interface DayPickerProps {
   captionElement?:
     | React.ReactElement<Partial<CaptionElementProps>>
     | React.ComponentClass<CaptionElementProps>
-    | React.SFC<CaptionElementProps>;
+    | React.FC<CaptionElementProps>;
   className?: string;
   classNames?: ClassNames;
   containerProps?: React.DetailedHTMLProps<
@@ -69,7 +69,7 @@ export interface DayPickerProps {
   navbarElement?:
     | React.ReactElement<Partial<NavbarElementProps>>
     | React.ComponentClass<NavbarElementProps>
-    | React.SFC<NavbarElementProps>;
+    | React.FC<NavbarElementProps>;
   numberOfMonths?: number;
   onBlur?: (e: React.FocusEvent<HTMLDivElement>) => void;
   onCaptionClick?: (month: Date, e: React.MouseEvent<HTMLDivElement>) => void;
@@ -142,7 +142,7 @@ export interface DayPickerProps {
   weekdayElement?:
     | React.ReactElement<Partial<WeekdayElementProps>>
     | React.ComponentClass<WeekdayElementProps>
-    | React.SFC<WeekdayElementProps>;
+    | React.FC<WeekdayElementProps>;
   weekdaysLong?: string[];
   weekdaysShort?: string[];
   tabIndex?: number;
